src/controller/app/app.controller.ts

```diff
@@ -35,10 +40,19 @@ export class AppController {
 
   @Get('health')
   @HealthCheck()
+  @CacheTTL(5000) // 5 seconds
```

Comment:
5 sec cache still gives us 31 days * 24 h/day * 3600 sec/h / 5 checks/s = 535 680 checks per month
Token is only valid for 50k/month, so this won't help for rate limiting issues.

Reply:
right - should I remove this ip info token from healthcheck?

Reply:
How about more aggressive caching, but not for whole healthcheck endpoint, just for ipinfo check? So remove caching from /healthcheck, but only pingCheck 'ipinfo.io' once an hour or so.

But removing ipinfo completely from healthcheck should be valid as well. When it starts failing, it will fail generating client reports and we have separate metrics that will alert us about that, so it won't go unnoticed. And at the same time it's not the kind of issue that will be solved by auto-failover.

Do whichever you prefer.

Reply:
we can check for `https://ipinfo.io`, instead of `https://ipinfo.io/8.8.8.8?token=${this.configService.get<string>('IP_INFO_TOKEN')}`, - I dont think this will trigger rate limiting

Reply:
Then it won't tell us that our token is no longer useful for actual work, so we can just as well remove it completely?

Reply:
yea its not perfect - ok, I'll add separate cache for ipinfo healthcheck